vapi/tags/tag_association.go

```diff
@@ -45,7 +45,10 @@ func (c *Manager) AttachTag(ctx context.Context, tagID string, ref mo.Reference)
 	}
 	spec := internal.NewAssociation(ref)
 	url := c.Resource(internal.AssociationPath).WithID(id).WithAction("attach")
-	return c.Do(ctx, url.Request(http.MethodPost, spec), nil)
```

Comment:
just returning an error, should be fine as-is

Reply:
Resolved in https://github.com/vmware/govmomi/commit/d14cbf39ee092c1a12388ab5b8009bc3c443c898.